chainermn/communicators/_base.py

```diff
@@ -87,7 +87,12 @@ def send(self, obj, dest, tag):
             'chainermn.communicators.CommunicatorBase.send')
 
         msgtype = _MessageType(obj)
-        self.mpi_comm.send(msgtype, dest=dest, tag=tag)
+        """We use ssend instead of send to pass unittest.
+        If we not use it, a error occurs in
```

Comment:
We use ssend() instead of send to pass unittests. If we don't use it, an error occurs in

Reply:
Thank you for your comment. I will fix it.